scss/mixins/typo.scss

```
@@ -0,0 +1,121 @@
+@font-face {
+    font-family: "SenBold";
+    src:   url("/public/fonts/Sen-Bold.ttf") format("truetype");
+    font-weight: bold;
+    font-style: normal;
+}
+@font-face {
+    font-family: "SenRegular";
+    src: url("/public/fonts/Sen-Regular.ttf") format("truetype");
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+@font-face {
+    font-family: "Inter";
+    src: url("/public/fonts/Inter-Regular.ttf") format("truetype"); 
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+@mixin fontSen{
+    font-family: "Sen",sans-serif;
+}
+
+@mixin fontInter {
+    font-family: "Inter",sans-serif;
+}
+@mixin heading1{
```

Chỗ này em có thể tối ưu hơn nữa bằng cách viết một cái mixin fontStyle cho các thuộc tính của font rồi @include vào, nó sẽ gọn hơn rất nhiều, áp dụng tương tự những thằng bên dưới
```suggestion
    @include fontSen;
    @include fontStyle($fontStyle: normal, $fontWeight: 700, $fontSize: 5.6rem, $lineHeight6.4rem, $letterSpacing: 0.2rem, $textColor: $color)
```